charts/console/values.yaml

```diff
@@ -234,6 +234,13 @@ secret:
       # tlsCert:
       # tlsKey:
 
+# -- Settings for license key, as an alternative to secret.enterprise when
+# a license secret is available
```

Comment:
NIT:

```suggestion
# -- Settings for Redpanda license, as an alternative to secret.enterprise because
# the secret.enterprise would be stored in helm values as plain text.
```